.github/pull_request_template.md

```diff
@@ -2,12 +2,20 @@
 
 > Why this pull request was created? Explain the problem that this pull request is trying to solve.
 
+[comment]: # (Add explanation bellow this line)
+
+
```

Comment:
The markdownlint does not like 2 empty lines in a row. More: https://github.com/DavidAnson/markdownlint/blob/main/doc/Rules.md#md012

```
MD012/no-multiple-blanks: Multiple consecutive blank lines [Expected: 1; Actual: 2]
```

Reply:
In my opinion it is a `Won't Fix` issue. Blank lines are added intentionally to make people's lives easier.

Reply:
Not sure how it makes people's lives easier. Adding 3 lines could make it easier as someone could just edit the line in the middle.

Reply:
There is no need to put a blank line under the comment, so it works the same as 3 lines. Markdown, btw. doesn't need a blank line between text and headers, so all these three works the same:
```
[comment]: # (Comment)

inserted by user

## Header
```
```
[comment]: # (Comment)
inserted by user

## Header
```
```
[comment]: # (Comment)
inserted by user
## Header
```

The added comments say:
> Add explanation bellow this line

so I added two blank lines, to have this:
```
[comment]: # (Add explanation bellow this line)


## Header
```
which edited by a developer looks like:
```
[comment]: # (Add explanation bellow this line)
I created this PR because it is very useful.

## Header
```

Reply:
we can configure the linter to accept 2 blank lines instead of 1. see: https://github.com/DavidAnson/markdownlint/blob/main/doc/Rules.md#md012---multiple-consecutive-blank-lines

I have created a PR to have dedicated rules for GH templates: https://github.com/goyek/goyek/pull/96